plugins/Jobs.cpp

```diff
@@ -49,7 +49,9 @@ BedrockJobsCommand::BedrockJobsCommand(SQLiteCommand&& baseCommand, BedrockPlugi
 {
 }
 
-BedrockPlugin_Jobs::BedrockPlugin_Jobs(BedrockServer& s) : BedrockPlugin(s)
+BedrockPlugin_Jobs::BedrockPlugin_Jobs(BedrockServer& s) :
+    BedrockPlugin(s),
+    isLive(server.args.isSet("-live"))
```

Comment:
did we update the bedrock startup script to include this arg?

Reply:
I have not. I was given the impression it's already there: https://expensify.slack.com/archives/C03TQ48KC/p1594597768212400

Reply:
https://github.com/Expensify/Salt/blob/master/shared_pkgs/bedrock/files/webrock-bedrock.service#L35-L36